src/basics.md

```diff
@@ -502,6 +503,57 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-continuous-process-output]: #ex-continuous-process-output
+<a name="ex-continuous-process-output"></a>
+## Continuously process child process' outputs
+
+[![std-badge]][std] [![cat-os-badge]][cat-os]
+
+Contrary to [Run an external command and process
+stdout](#ex-parse-subprocess-output), while an external [`Command`] is
+running, process its standard output. A new pipe is created by
+[`Stdio::piped`] and it is read by a [`BufReader`]. Note, a buffered
+reader uses a buffer to reduce the number of I/O requests.
```

Comment:
I'm not sure that we need to explain what `BufReader` does

Reply:
True, it's slightly off topic. While I was implementing the example, I ask myself the question `std::io::BufReader` vs `std::io::Reader`. I will remove it. Plus the [`BufReader`](https://doc.rust-lang.org/std/io/struct.BufReader.html) link to the doc explains it:

> It can be excessively inefficient to work directly with a Read instance. For example, every call to read on TcpStream results in a system call. A BufReader performs large, infrequent reads on the underlying Read and maintains an in-memory buffer of the results.